contracts/lease/src/lease/mod.rs

```diff
@@ -112,6 +115,9 @@ where
     LppLoan::Error: Into<ContractError>,
     Oracle: OracleTrait<Lpn>,
 {
+    // const MIN_ASSET: LpnCoin = LpnCoin::from(Coin::<Lpn>::new(10_000));
+    // const MIN_SELL_ASSET: LpnCoin = LpnCoin::from(Coin::<Lpn>::new(15_000_000));
+
```

Comment:
move in PositionDTO and use

Reply:
I'm not sure this is a good idea since those const-s are used in lease/mod.rs and in lease/liquidation.rs

Reply:
ok but when you start using `PositionSpec` in the domain model they should become unused